portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -122,4 +128,20 @@ private BlobKey getBlobKey(List<BlobKey> blobKeys, BlobstoreService blobstoreSer
     }
     return blobKey;
   }
+
+  /** Returns the sentiment score of the given comment, or 0 if the user didn't wrote a comment. */
+  private float getSentimentScore(String comment) throws IOException {
+    if (comment == null || comment.equals("")) {
+      return 0;
```

Comment:
[Optional] It might be better to use a value that can't be a valid score, specifically Float.NaN.  However, if you want to treat the empty comment as neutral sentiment I think it's find to use 0 here.

Reply:
I'm displaying empty comments as neutral.